package.json

```diff
@@ -13,7 +13,7 @@
     "test:lint": "semistandard --verbose | snazzy",
     "test:cover": "nyc --reporter=text-summary --reporter=html npm test",
     "test:coveralls": "nyc --reporter=text-lcov npm test | coveralls",
-    "test:all": "npm run test:lint"
+    "test:all": "npm run test:lint && npm run test && npm run test:cover && npm run test:coveralls"
```

Comment:
Here it should be:
```js
"test:all": "npm run test:lint && npm run test:cover"
```

Since `test:cover` already run `npm test` and `test:coveralls` is meant only for the CI to publish on coveralls